CMakeLists.txt

```diff
@@ -10,6 +10,50 @@ and Nvidia GPUs for hardware encoding. Software encoding is also available. You
 Moonlight client on a variety of devices. A web UI is provided to allow configuration, and client pairing, from \
 your favorite web browser. Pair from the local server or any mobile device.")
 
+# Generate Sunshine Version based of the git tag
+# https://github.com/nocnokneo/cmake-git-versioning-example/blob/master/LICENSE
+find_package(Git)
+if(GIT_EXECUTABLE)
+    MESSAGE("${CMAKE_CURRENT_SOURCE_DIR}")
+    get_filename_component(SRC_DIR ${CMAKE_CURRENT_SOURCE_DIR} DIRECTORY)
+    #Get current Branch
+    execute_process(
+        COMMAND ${GIT_EXECUTABLE} rev-parse --abbrev-ref HEAD 
+        #WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+        OUTPUT_VARIABLE GIT_DESCRIBE_BRANCH
+        RESULT_VARIABLE GIT_DESCRIBE_ERROR_CODE
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    # Gather current commit
+    execute_process(
+        COMMAND ${GIT_EXECUTABLE} rev-parse --short HEAD
+        #WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+        OUTPUT_VARIABLE GIT_DESCRIBE_VERSION
+        RESULT_VARIABLE GIT_DESCRIBE_ERROR_CODE
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    # Check if Dirty
+    execute_process(
+        COMMAND ${GIT_EXECUTABLE} diff --quiet --exit-code
+        #WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
+        RESULT_VARIABLE GIT_IS_DIRTY
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+    if(NOT GIT_DESCRIBE_ERROR_CODE)
+        MESSAGE(Sunshine Branch: ${GIT_DESCRIBE_BRANCH})
+        if(NOT GIT_DESCRIBE_BRANCH STREQUAL "master")
+            set(PROJECT_VERSION ${PROJECT_VERSION}-g${GIT_DESCRIBE_VERSION})
```

Comment:
I think this (and L49) is causing the error with the AUR - https://github.com/LizardByte/Sunshine/actions/runs/4091756865/jobs/7056202851#step:7:88

What if we replace the `-` with a `.`?

Then probably don't set the subversion for the AUR as we currently do? Would also require a change here in these places:

-https://github.com/LizardByte/Sunshine/blob/f8819d32e324a02421fe838baadd000210b61d4a/packaging/linux/aur/PKGBUILD#L5
- https://github.com/LizardByte/Sunshine/blob/f8819d32e324a02421fe838baadd000210b61d4a/.github/workflows/CI.yml#L171
- https://github.com/LizardByte/Sunshine/blob/f8819d32e324a02421fe838baadd000210b61d4a/.github/workflows/CI.yml#L191
- https://github.com/LizardByte/Sunshine/blob/f8819d32e324a02421fe838baadd000210b61d4a/.github/workflows/CI.yml#L198
- https://github.com/LizardByte/Sunshine/blob/f8819d32e324a02421fe838baadd000210b61d4a/.github/workflows/CI.yml#L212

Subversion on AUR was more to handle nightly releases, which we're not doing currently anyway.